FrameworkSystem/Client/ProxyManagerClient.py

```diff
@@ -475,6 +476,16 @@ def deleteGeneratedProxyFile(self, chain):
     self.__filesCache.delete(chain)
     return S_OK()
 
+  def deleteProxyBundle(self, idList):
+    """ delete a list of id's
+
+        :param list,tuple idList: list of identity numbers
+
+        :return: S_OK(int)/S_ERROR()
+    """
+    rpcClient = RPCClient("Framework/ProxyManager", timeout=120)
```

Comment:
should be `_getRPC` no ?

Reply:
For a reason unknown to me (a good one? not sure) this `ProxyManagerClient` does not inherit from `Client`, so there's no `_getRPC`). That's why I am exposing this RPC call directly here.